src/Microsoft.DotNet.Cli.Utils/FrameworkDependencyFile.cs

```diff
@@ -45,6 +46,36 @@ public string GetNetStandardLibraryVersion()
                 ?.Version;
         }
 
+        public bool TryGetMostFitRuntimeIdentifier(IEnumerable<string> candidateRuntimeIdentifiers, out string mostFitRuntimeIdentifier)
+        {
+            mostFitRuntimeIdentifier = null;
+
+            if (!SupportsCurrentRuntime())
```

Comment:
(perf nit) Checking to see if the RID is in the graph first, and then searching again to find it seems unnecessary to me.  You can just do a single search to find it and return 'false' if it isn't found.